src/test/groovy/org/kohsuke/groovy/sandbox/TheTest.groovy

```diff
@@ -316,7 +316,7 @@ x.plusOne(5)
      * See issue #9
      */
     void testAnd() {
-        assertIntercept('Checker:checkedCast(Class,null,Boolean,Boolean,Boolean)', false, """
+        assertIntercept('', false, """
```

Comment:
This suggests to me that [this entry](https://github.com/jenkinsci/script-security-plugin/blob/ea5ce369c4ed064c64be9ac93a03c12e08d5d167/src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist#L498-L499) may be obsolete.